cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -5224,6 +5224,45 @@ TEST(TestArrowReadWrite, WriteAndReadRecordBatch) {
   EXPECT_TRUE(record_batch->Equals(*read_record_batch));
 }
 
+TEST(TestArrowReadWrite, WriteRecordBatchNotProduceEmptyRowGroup) {
```

Comment:
Can you add comment explaining what this tests?

Reply:
done